nextcloud/user_saml
# allow IDP to initiate login

if the user starts the login process on the IDP, we always assume that
the first configured IDP is used (this doesn't work for multiple IDP's
for now) and tries to login the user.

Steps to test:

- Setup an IDP, e.g. onelogin
- Go directly to the IDP (not to the Nextcloud)
- Select there the "Nextcloud app"
- You will be redirected to Nextcloud

-> Current behavior: You will see an error page
-> with this PR: you should be logged in
